python/bifrost/udp_transmit.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from libbifrost import _bf, _check, _get
+
+import ctypes
+import numpy as np
+
+def _packet2pointer(packet):
+	buf = ctypes.c_char_p(packet)
+	siz = ctypes.c_uint( len(packet) )
+	return buf, siz
+
+
+def _packets2pointer(packets):
+	count = ctypes.c_uint( len(packets) )
+	buf = ctypes.c_char_p("".join(packets))
```

Comment:
For a first pass at this I wanted something that was very generic and did not make too many assumptions about the packets (this only assumes that all of the packets are the same size).  In the future it would be nice to have something similar to what you suggested but with the classes creating the headers on-the-fly depending on the packet format.